site/sfguides/src/kepler_gl_maps_inside_snowflake/kepler_gl_maps_inside_snowflake.md

```diff
@@ -0,0 +1,225 @@
+
+1. **Navigate to the Marketplace** in Snowsight.
+2. **Search for "Dekart"**
+
+![Install Dekart for Snowflake](assets/kepler_gl_snowflake_3.png)
+
+3. Start a **30-day free trial**.
+
+**Note**: Dekart is open-source, meaning you can opt for a self-hosted setup if preferred. Once configured, Dekart runs as a Snowpark Container Application within your Snowflake account, ensuring security and direct data access.
+
+![Dekart Free Trial](assets/kepler_gl_snowflake_4.png)
+
+
+4. Follow the installation instructions as displayed in the Snowflake interface.
+
+![Dekart Installation Instructions](assets/kepler_gl_snowflake_5.png)
+
+5. Grant Account Privileges to Dekart.
+
+**Note**: Dekart creates single node `CPU_X64_XS` copmute pool and `XSMALL` warehouse.
+
+![Grant Permissions to Dekart](assets/kepler_gl_snowflake_6.png)
+
+6. Allow connections to the Mapbox API.
+
+Dekart uses Mapbox for rendering maps. No user data is sent to Mapbox.
+
+![Allow Mapbox API Connections](assets/kepler_gl_snowflake_7.png)
+
+7. Activate Dekart.
+
+This step may take around 10 minutes on Snowflake’s end.
+
+![Activate Dekart](assets/kepler_gl_snowflake_8.png)
+
+
+### Step-by-Step Video Guide
+
+Watch this video for a detailed walkthrough of the Dekart installation process.
+
+<video id="KusNayeGFaI"></video>
+
+
+<!-- ------------------------ -->
+## Access Overture Maps
+Duration: 2
+
+To build our map, we need geospatial datasets from Overture Maps, available in the Snowflake Marketplace.
+
+1. **Go to Snowsight** (Snowflake’s web interface) and select **Marketplace**.
+2. **Search for "Overture Maps"** to find geospatial datasets.
+3. Add **Overture Maps Divisions, Places, and Transportation datasets** to your Snowflake account.
+
+   - These datasets include information on administrative divisions, transportation routes, and points of interest. The [Overture Maps Schema Reference](https://docs.overturemaps.org/schema/reference/) is an excellent resource to understand the structure and details of each dataset.
+
+**Note**: You’ll need access to these tables to use them within Snowflake. Adding them from the Marketplace will ensure they’re ready for querying.
+
+![Access Free Overture Maps Data](assets/kepler_gl_snowflake_2.png)
+
+
+<!-- ------------------------ -->
+## Configure Permissions to Data
+Duration: 2
+
+Since Dekart operates within your Snowflake account, it requires permissions to access the Overture Maps datasets. This ensures that Dekart can read and visualize the data within Snowflake without transferring it externally.
+
+Execute the following SQL commands in Snowflake (make sure you have the `ACCOUNTADMIN` role for these operations):
+
+```sql
+-- Run as ACCOUNTADMIN
+GRANT IMPORTED PRIVILEGES ON DATABASE OVERTURE_MAPS__TRANSPORTATION TO application DEKART__WEBGL_MAPS_FOR_SNOWFLAKE;
```

Comment:
It seems the correct name of the app is `DEKART__KEPLER_GL_MAPS_INSIDE_SNOWFLAKE`